provider/cmd/pulumi-resource-aws/bridge-metadata.json

```diff
@@ -220172,11 +220164,6 @@
             "aws_vpc_ipam_pool": {
                 "current": "aws:ec2/getVpcIpamPool:getVpcIpamPool",
                 "past": [
-                    {
```

Comment:
{
                        "name": "aws:ec2/getVpcIamPool:getVpcIamPool",
                        "inCodegen": false,
                        "majorVersion": 6
                    },
                    {
                        "name": "aws:ec2/getVpcIamPool:getVpcIamPool",
                        "inCodegen": false,
                        "majorVersion": 6
                    }

There were two identical entries here, so this is a reasonable change.